.vscode/settings.json

```diff
@@ -2,8 +2,6 @@
   "editor.rulers": [80],
   "files.exclude": {
     "**/.git": true,
-    "**/node_modules": true,
```

Comment:
what does this do? (I don't use vs code)

Reply:
ah, just saw

>As a side note hiding all the build output from VScode is a pain, and again falls into the category of things I consider lying to developers. It wasted a bunch of my time because I couldn't tell that typescript watching was not working.